src/snowflake/snowpark/session.py

```diff
@@ -19,7 +18,7 @@
 from typing import Any, Dict, List, Literal, Optional, Sequence, Set, Tuple, Union
 
 import cloudpickle
-from packaging.requirements import Requirement
```

Comment:
My guess is that it is caused by this line, though it's unclear to me how solver can be impacted 🤦